src/ascent/runtimes/flow_filters/ascent_runtime_babelflow_pmt.cpp

```diff
@@ -674,14 +1016,25 @@ void ascent::runtime::filters::BFlowPmt::execute()
     int64_t fanin = p["fanin"].as_int64();
     FunctionType threshold = p["threshold"].as_float64();
     int64_t gen_field = p["gen_segment"].as_int64();
+    int64_t gen_rel = p["rel_field"].as_int64();
```

Comment:
We now have updated containers, -- looks like this  line is failing during the unit test runs

```
77: Node::as_int64() const -- DataType empty at path rel_field does not equal expected DataType int64
```

full log:
[2021_12_blfow_run_log.txt](https://github.com/Alpine-DAV/ascent/files/7794394/2021_12_blfow_run_log.txt)

Reply:
@cyrush It's a good point - the bug is that `rel_field` is an optional parameter and I don't check whether it's defined or not. I pushed a fix now.